below/model/src/collector_plugin.rs

```diff
@@ -155,14 +155,14 @@ mod test {
             // Test overwriting sample
             futures::executor::block_on(collector.collect_and_update()).unwrap();
             c.wait(); // <-- 1
-            // Consumer checking overwritten sample
+                      // Consumer checking overwritten sample
```

Comment:
nit: I actually think this made more sense before the rustfmt. I think the full line comments were for the statements after.

Reply:
thanks for noticing, weird change by `rustfmt`!